packages/core/saved-objects/core-saved-objects-migration-server-internal/src/migrations_state_action_machine.test.ts

```diff
@@ -98,7 +99,7 @@ describe('migrationsStateActionMachine', () => {
       abort,
     });
     const logs = loggingSystemMock.collect(mockLogger);
-    const doneLog = logs.info.splice(8, 1)[0][0];
+    const doneLog = logs.info.splice(4, 1)[0][0];
```

Comment:
NIT could you add a little comment describing what you're doing with this obscure line?

Reply:
`I`'m doing nothing, just adapting `someone else`'s hack :trollface: